extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -108,3 +79,39 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 
 	return true;
 }
+
+function fetchReviewHistory( $dbr, $cityId, $pageId ) {
+	// TODO: fetch history from image
+
+	$res = $dbr->select(
+		'image_review_stats',
+		'*',
+		[
+			'wiki_id' => $cityId,
+			'page_id' => $pageId,
+		]
+	);
+
+	$reveiws = [];
+	while ( $row = $dbr->fetchObject( $res ) ) {
+		$data = [];
+		$data[] = User::newFromId( $row->reviewer_id )->getName();
+		$data[] = wfMsg( 'imagereview-state-' . $row->review_state );
+		$data[] = $row->review_end . ' (UTC)';
+
+		$reviews[] = $data;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$reviews" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)